modules/nf-httpfs/src/main/nextflow/file/http/LatchFileSystem.groovy

```diff
@@ -4,16 +4,26 @@ import java.nio.file.Path
 
 import groovy.util.logging.Slf4j
 
+import java.util.concurrent.ForkJoinPool
```

Comment:
would throw this in FileSystemProvider bc iirc there can be more than 1 FileSystem

Reply:
shouldnt matter tho